File: src/app/models/Employe.ts
import { TypeEmploye } from './TypeEmploye';
import { EtatEmploye } from './EtatEmploye';
export class Employe {
  constructor(
    public idemploye :number,
    public etate:string,
    public datedenaissance:Date ,
    public nom: string,
    public  prenom: string,
  
    public adresseemploye:String  ,
    public montantavantage:number ,
    public montantsalaire:number ,
    public  password :String,
    public typeEmploye: string,
    
    
  ){}

}

File: src/main/java/com/project/un_site_de_gestion/entities/Employe.java
package com.project.un_site_de_gestion.entities;

import lombok.*;

import javax.persistence.*;
import java.util.Date;

@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Builder
public class Employe {
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private int idEmploye ;
    @Enumerated(EnumType.STRING)
    private Etat etat;
    private Date DateDeNaissance;
    private String nom ;
    private String prenom;
    private String  adresseEmploye;
}
